src/app/credExplorer/markovChain.test.js

```diff
@@ -77,3 +80,19 @@ describe("sparseMarkovChainFromTransitionMatrix", () => {
     );
   });
 });
+
+describe("uniformDistribution", () => {
+  it("computes the uniform distribution with domain of size 1", () => {
+    const pi = uniformDistribution(1);
+    expect(pi).toEqual(new Float64Array([1]));
+  });
+  it("computes the uniform distribution with domain of size 4", () => {
+    const pi = uniformDistribution(4);
+    expect(pi).toEqual(new Float64Array([0.25, 0.25, 0.25, 0.25]));
+  });
+  [0, -1, Infinity, NaN, 3.5, '"beluga"', null, undefined].forEach((bad) => {
```

Comment:
their Magnificence, the Magnificent Foo Plugin, sees your wisdom in not including them in this test case. for, to do so would be to shortly assign their Magnificence to a variable named `bad`, for which the consequences would be ... grave

Reply:
I humbly agree.